packages/appstore/src/components/modals/real-wallets-upgrade/real-wallets-upgrade.tsx

```diff
@@ -0,0 +1,171 @@
+import React from 'react';
+import { DesktopWrapper, MobileDialog, MobileWrapper, Modal, Button } from '@deriv/components';
+import { useWalletMigration } from '@deriv/hooks';
+import { ContentFlag } from '@deriv/shared';
+import { localize } from '@deriv/translations';
+import { observer, useStore } from '@deriv/stores';
+import { WalletsIntro } from './components/wallets-intro/wallets-intro';
+import ReadyToUpgradeWallets from './components/ready-to-upgrade-wallets';
+import './real-wallets-upgrade.scss';
+import WalletLinkingStep from './wallet-linking-step/wallet-linking-step';
+import getMockWalletMigrationResponse from '../../../constants/mock_wallet_migration_response';
```

Comment:
could we add constants as a relative path? so it can be like this: `Constants/...`